applications/tari_console_wallet/src/automation/command_parser.rs

```diff
@@ -318,9 +318,9 @@ fn parse_make_it_rain(mut args: SplitWhitespace) -> Result<Vec<ParsedArgument>,
     // txs per second
     let txps = args.next().ok_or_else(|| ParseError::Empty("Txs/s".to_string()))?;
     let txps = txps.parse::<f64>().map_err(ParseError::Float)?;
-    if txps > 25.0 {
-        println!("Maximum transaction rate is 25/sec");
-        return Err(ParseError::Invalid("Maximum transaction rate is 25/sec".to_string()));
+    if txps > 2500.0 {
+        println!("Maximum transaction rate is 2500/sec");
+        return Err(ParseError::Invalid("Maximum transaction rate is 2500/sec".to_string()));
```

Comment:
**Discussion**: The 25 txn/s maximum rate limit was added due to instabilities in the Tokio environment. This may or may not be an issue anymore; I have not tested it again since the limit was introduced. Would be interesting to experiment with a 2500 txn/s transaction instantiation rate.